Content/RecipeChanges/FoxRecipeChanges/Calamity.cs

```diff
@@ -0,0 +1,33 @@
+using CalamityMod.Items.Materials;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using ThoriumMod.Items.BardItems;
+using ThoriumMod.Items.HealerItems;
+using ThoriumMod.Items.ThrownItems;
+
+namespace TheBereftSouls.Content.RecipeChanges;
+
+[ExtendsFromMod("CalamityMod", "SOTS", "ThoriumMod")]
+public class Calamity : ModSystem
+{
+    public override void PostAddRecipes()
+    {
+        foreach (var recipe in Main.recipe)
+        {
+            if (recipe.createItem.type == ModContent.ItemType<GalacticaSingularity>())
+                recipe.DisableRecipe();
+        }
+        Recipe newRecipe = Recipe.Create(ModContent.ItemType<GalacticaSingularity>(), 2);
```

Comment:
Could clone Galactica Singularityrecipe and append new ingredients on it. Instead of just disabling it.